### DISCUSSION TITLE
Quest 2 Games

### DISCUSSION THREAD

**Discussion by KVonGit** [textadventures/quest#1296] [Libraries and Code Samples]
All 28 of the Quest 2 games on the site. [^1]

[Quest 2 Games.pdf](https://github.com/user-attachments/files/18219583/Quest.2.Games.pdf)

---
Title | Author | Link | Genre | Platform
-- | -- | -- | -- | --
Adventures of Koww the Magician, The | Brian the Great | https://textadventures.co.uk/games/view/nalydifu9eqbcgy0pcyceq | Fantasy | Quest 2.0
Black Forest Part 1: Winterside | Devon Oratz | https://textadventures.co.uk/games/view/udcark5eaekctkzbvnvlrw | Fantasy | Quest 2.17
Certain Oscar, A | Ido Angel | https://textadventures.co.uk/games/view/qzuz5wnuzkyr1qetoo_tig | Mystery | Quest 2.17
Devil’s Bargain, The | Chris Groskopf | https://textadventures.co.uk/games/view/pcklrtjsgucdhplslps42w | RPG | Quest 2.14
Dream Weaver, The | Bad Omen | https://textadventures.co.uk/games/view/rk20y5qc70mo-mzvjtgjtg | Fantasy | Quest 2.17
Easy Money | Kirk Karanza | https://textadventures.co.uk/games/view/nrg-7axymeyojawccg82la | Educational | Quest 2.1
Fade to White | Devon Oratz | https://textadventures.co.uk/games/view/fd2oe2_evei7xs4wvb23bw | Fantasy | Quest 2.1
Magic Sword Part 1: Legacy of Earth | dom | https://textadventures.co.uk/games/view/ck35po5o70yv6py-ztgoxa | RPG | Quest 2.17
Space: The Final F*** Up | Stretch | https://textadventures.co.uk/games/view/_gh9akjvlu6g888outu0pg | Sci-Fi Adult | Quest 2.1
Uranus or Bust! Episode 1: Escort | Sam Bickley | https://textadventures.co.uk/games/view/hund693ffeqttnepirll8g | Sci-Fi | Quest 2.17
Venus Flytrap: Romantic Music | Nicky Leech | https://textadventures.co.uk/games/view/fm_pitfz6uudiaki62vgqa | Romance | Quest 2.1
Black Out | Eric Strong | https://textadventures.co.uk/games/view/1wo8yl6khu21pfgkwualtq | Sandpit | Quest 2.17
Catacombs | Eric Strong | https://textadventures.co.uk/games/view/kv7iwats4021eeko2et2jq | Sandpit | Quest 2.17
Cheese Adventures | The Mighty Achoo | https://textadventures.co.uk/games/view/hkbql2sxu0clc6vexh2pyq | Sandpit | Quest 2.17
Dark Realm | Stephan Christensen | https://textadventures.co.uk/games/view/nukxdkskzkqlhelthhcuma | Sandpit | Quest 2.1
Earthquake! And Earthquake 2! | Brad Westlake | https://textadventures.co.uk/games/view/d-ror9q9hek-ytq9kauvdg | Sandpit | Quest 2.17
Evil Work | Blaine Sim | https://textadventures.co.uk/games/view/zqdu3aeobuchk9utlr2wta | Sandpit | Quest 2.17
Hobbit: Vol I, The | nplindsell | https://textadventures.co.uk/games/view/p-bmq3zuake8zuem2cu1oa | Sandpit | Quest 2.1
House | Kris Cyou | https://textadventures.co.uk/games/view/izuc6vt6n0cuhdtnsejsgg | Sandpit | Quest 2.1
Legend of the Elements (Demo v2.0), The | Jimmy Kilpatrick | https://textadventures.co.uk/games/view/qzuz5wnuzkyr1qetoo_tig | Sandpit | Quest 2.1
Love Song | Mihalis "DarkAng3l" Georgostathis | https://textadventures.co.uk/games/view/nwa1gfiyveaf6pqa8bo_ya | Sandpit | Quest 2.1
Lumberjack’s Quest | Phil Barker | https://textadventures.co.uk/games/view/tohd-9dwxu_kv6jxtq6d_q | Sandpit | Quest 2.1
Middle-Earth (full version) | QuestHobbit | https://textadventures.co.uk/games/view/002bpjsbme_chhyptn6cfw | Sandpit | Quest 2.17
Missing Dog | Emily Strong | https://textadventures.co.uk/games/view/cceaj76_eea6w80atxgidw | Sandpit | Quest 2.17
Omega: Code Nine | Chris Walker | https://textadventures.co.uk/games/view/-kwerhc-cew28ov8q_8h4q | Sandpit | Quest 2.1
Robbers | Scott Warner | https://textadventures.co.uk/games/view/xasawtihp0qjggzkejkatq | Sandpit | Quest 2.1
Silent Rage (Demo) | Rick Greenlee | https://textadventures.co.uk/games/view/2lz00z9dpewtw05zuskiug | Sandpit | Quest 2.1
Weird World of Sam Rogers, The | Sam Rogers | https://textadventures.co.uk/games/view/s16vtzljyeyhjo5b0vz7wg | Sandpit | Quest 2.1

[^1]: https://discord.com/channels/1278365322054996018/1314182688734711898/1320067146843295865

**Comment by David-Jareth-Bowie** [comment]
I did not even know we had a Quest 2... 🤔

**Reply by Dragonoftheworld** [reply]
wait till you hear we have Quest 5 😁

**Reply by KVonGit** [reply]
Yeah, it was 1999-2001. I just recently 'discovered' the older versions.

07/NOV/1998 - QUEST 1.0
11/AUG/1999 - QUEST 2.0
08/SEP/2001 - QUEST 3.0
17/JAN/2007 - QUEST 4.0
30/AUG/2011 - QUEST 5.0

Quest 5 was a complete rewrite: https://blog.textadventures.co.uk/2011/08/30/quest-5-0-is-out-now/

**Reply by David-Jareth-Bowie** [reply]
Oh... I see.  Haha! 😅  So you can upload older versions of Quest and they will play in their entirety?  I was close to completion of one game, an update broke certain aspects of it and I left it at that.  Could it start working again?

**Reply by KVonGit** [reply]
> So you can upload older versions of Quest and they will play in their entirety?

Mostly. Some of the games from before like Quest 5.4 had a couple of features that won't work, but it's just little stuff like displaying a picture in its own window (and now the picture will just be inserted in the game's text).

**EDIT:**
I haven't seen any games that won't work. The online player will just ignore some audio formats (like MIDI, because browsers don't play those natively), and there's that one thing with the images. Nothing effects game-play.


---
> I was close to completion of one game, an update broke certain aspects of it and I left it at that. Could it start working again?

I've been helping people restore Quest games recently, and I've been using the same version of Quest they used. The last one was a 5.6.3 game. So, I uninstalled 5.8 and installed 5.6.3 to fix the bugs he'd recently found.

Note that the ASL version changed in 5.8.

The second line of code in your ASLX file in 5.8 will be: `<asl version="580">`

In 5.6.3, that was: `<asl version="550">`

If you edited your game and saved in 5.8, it changed that line to `580`. To open the game for editing in 5.6.3 (if that is in fact the version you need), you'd need to open that ASLX file in a text editor and change `580` to `550` before Quest will open the file for editing.

---
If you end up trying this and run into any problems, just let us know. We should be able to help.

**Reply by David-Jareth-Bowie** [reply]
I appreciate that.  The issue was hand made code by someone in the community.  I needed a certain task and they did it wonderfully.  But then the next version broke it.  Could hand made code work with what you were mentioning?

**Reply by KVonGit** [reply]
I want to say yes, but I guess it really depends on what's in the code. If it worked in 5.7.2, then 5.8 broke it, it can probably be fixed by switching back to 5.7.2.

It depends on whether any code was changed after switching to 5.8, too, I guess.

Either way, I would bet it can be fixed.
